templates/bootstrap/plugin_name/app/tasks/synchronizing.py.j2

```diff
@@ -93,4 +93,4 @@ class {{ plugin_camel_short }}FirstStage(Stage):
         Args:
             path: Path to the metadata file
         """
-        pass
+        return []
```

Comment:
not sure if we should return it, what was the error you saw here? I'm afraid returning an empty list will make harder for people to find where they need to implement

Reply:
A similar case to the previous comment. Users will not need to dig in the journals to see the error message and why Pulp even raised an error. These two changes make a plugin consistent with the documentation, meaning that the plugin will somehow work (as specified in the docs), but still provide reasonable initial behaviour where not all of the features are implemented.

Reply:
The error message was basically - `None object is not iterable` (or something like that), since it returned `None` (https://github.com/pulp/plugin_template/blob/74c9bb33c41051fe78321ff255413c1d7a31af9f/templates/bootstrap/plugin_name/app/tasks/synchronizing.py.j2#L76).

Reply:
got it! I was worried it would make it harder to find why it errored, but you are right!